package.json

```diff
@@ -16,5 +16,13 @@
     "react-native",
     "svg",
     "image"
-  ]
+  ],
+  "dependencies": {
+    "bluebird": "^3.5.1",
+    "debug": "^3.1.0",
+    "prop-types": "^15.6.2"
+  },
+  "devDependencies": {
+    "supports-color": "^5.4.0"
```

Comment:
This package doesn't seem to be used anywhere. Was this required for the use of `debug` package? Can this be removed?